src/core/dom/image/loader.ts

```diff
@@ -99,6 +100,26 @@ export default class ImageLoader {
 		this.cacheDefaultImage(this.defaultPreviewImageOptions, 'preview');
 	}
 
+	/**
+	 * Preprocess options for `ImageLoader`
```

Comment:
```suggestion
	 * Resolves the given operation options
```

+ Мне кажется, что это должен быть protected метод